### PR TITLE
docs: Add plugin  list and detail acquisition interface

### PR DESCRIPTION
**Pull request by qwxingzhe**
### What this PR does / why we need it:

Add plugin  list and detail acquisition interface

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

image

### CODE REVIEW

File: doc/admin-api.md [hunk 1]
```diff
@@ -714,16 +747,4 @@ Date: Thu, 26 Dec 2019 04:19:34 GMT
 Content-Type: text/plain
 ```
 
-[Back to TOC](#Table-of-Contents)
-
-*API*：/apisix/admin/plugins/?all=true
```
Comment (spacewander) [inline doc/admin-api.md:719]: The documentation show the behavior of the master branch code, not the release version. We are planning to maintain different documentation for different release in their branches. Anyway, the master branch of documentation should be corresponding with the master branch of code.

**Review by spacewander — CHANGES_REQUESTED** [review]

File: doc/admin-api.md (hunk 1 above)
Comment (qwxingzhe) [reply doc/admin-api.md:719]: I found it. I'll revise it

**Review by spacewander — APPROVED** [review]

**Review by tokers — APPROVED** [review]

File: doc/admin-api.md [hunk 2]
```diff
@@ -716,6 +717,33 @@ Content-Type: text/plain
...
+
+*API*：/apisix/admin/plugins/{plugin_name}
+
+*Description*: plugin
+
+> Request Methods:
+
+|Method      |Request URI|Request Body|Description        |
+|---------|-------------------------|--|------|
+|GET      |/apisix/admin/plugins/list|NULL|Fetch resource list|
```
Comment (juzhiyuan) [inline doc/admin-api.md:730]: @nic-chen @starsz Does manager-api supports this feature?

Comment (nic-chen) [reply doc/admin-api.md:730]: sure
`/apisix/admin/plugins` returns plugin name list like this.

Comment (juzhiyuan) [reply doc/admin-api.md:730]: ok

**Pull request merged**